### PR TITLE
tests added for dashboard messages

### PR DESCRIPTION
**Pull request by vishalpandeyvip**
Tests added for authorization and dashboard of fyle slack UI.

Imrovement
dashboard's coverage -- 30% to 95%,
Total coverage             -- 65% to 68%

Attesting the coverage report of files. 
Screenshot from 2022-11-10 16-34-41

### CODE REVIEW

File: tests/test_messages.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+FYLE_STOPLIGHT_URL = os.environ['FYLE_STOPLIGHT_URL']
+MOCK_FYLE_OAUTH_URL = FYLE_STOPLIGHT_URL
+MOCK_USERNAME = 'fake_username'
+
+class TestAuthorizationMessages:
+
+    def test_pre_authorization_message(self):
+        pre_authorization_message = authorization.messages.get_pre_authorization_message(MOCK_USERNAME, FYLE_STOPLIGHT_URL)
+        assert type(pre_authorization_message) is list
```
Comment (ashwin1111) [inline tests/test_messages.py:12]: can we store the expected return value as fixture and assert with it?

File: tests/test_messages.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    def test_post_authorization_message(self):
+        post_authorization_message = authorization.messages.get_post_authorization_message()
+        assert type(post_authorization_message) is list
+
+class TestDashboardMessages:
+
+    def test_get_pre_authorization_message(self):
+        pre_authorization_message = authorization.messages.get_pre_authorization_message(MOCK_USERNAME, MOCK_FYLE_OAUTH_URL)
+        assert 'type' in pre_authorization_message[0]
```
Comment (ashwin1111) [inline tests/test_messages.py:22]: same as 1st comment

File: tests/test_messages.py (hunk 1 above)
Comment (ashwin1111) [reply tests/test_messages.py:12]: if you think asserting the whole message will be overkill, we can just check specific parts of it, for example
`'text': 'Hey there *{}* :wave:'.format(user_name)`, you can check if text has the expected value

File: tests/test_messages.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        assert type(post_authorization_message) is list
+
+    def test_get_sent_back_reports_dashboard_view(self):
+        mock_reports = {
+            'total_amount': 100,
+            'count': 1,
+            'url': MOCK_FYLE_OAUTH_URL
+        }
+        sent_back_reports_view = dashboard.messages.get_sent_back_reports_dashboard_view(mock_reports, 'INR')
+        assert type(sent_back_reports_view) is list
```
Comment (ashwin1111) [inline tests/test_messages.py:35]: same, can assert important keys that involves amount, count, etc

File: tests/test_messages.py (hunk 1 above)
Comment (vishalpandeyvip) [reply tests/test_messages.py:12]: will this work
```
for text_object in pre_authorization_message: 
    assert 'text' in text_object
```

Comment (ashwin1111) [reply tests/test_messages.py:12]: `assert pre_authorization_message[0]['text'] == 'Hey there *fake_username* :wave:'`

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**